test/integration/dbt_project.yml

```diff
@@ -3,4 +3,8 @@ name: 'sqlserver_integration_tests'
 version: '1.0'
 config-version: 2
 
-profile: 'integration_tests'
\ No newline at end of file
+profile: 'integration_tests'
+
+models:
+  sqlserver_integration_tests:
+    +as_columnstore: false
```

Comment:
@jtcohen6 I thought this would be the right place for this but I'm still getting the error that this change was intended to fix:
> `'COLUMNSTORE'` is not supported in this service tier of the database.